example_tb/core/Makefile

```diff
@@ -99,10 +100,14 @@ vsim-all: .opt-rtl
 	$(RTLSRC_VOPT_TB_TOP)
 	touch .opt-rtl
 
+# fpnew dependencies
+fpnew_deps:
+	rm -rf ../../rtl/fpnew; git clone https://github.com/pulp-platform/fpnew ../../rtl/fpnew --recurse
```

Comment:
If you run multiple times this commands, it clones everytime the repo. Same when you execute multiple times the custom-fp program. Can the rm -rf part being part of the clean target rather than fpnew_deps?

Reply:
Hi @davideschiavone I changed the Makfile based on your comment; can you have another look please?